<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Karl Kunze" />
    <meta name="date" content="2023-05-05" />
    <title>Karl Kunze</title>

    
        <link href="Resume_files/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
        <link href="Resume_files/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />
        <link href="Resume_files/paged-0.20/css/resume.css" rel="stylesheet" />
        <script src="Resume_files/paged-0.20/js/config.js"></script>
        <script src="Resume_files/paged-0.20/js/paged.js"></script>
        <script src="Resume_files/paged-0.20/js/hooks.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:khk44@cornell.edu" class="email">khk44@cornell.edu</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/karlkunze">github.com/karlkunze</a></li>
<li><i class="fa fa-phone"></i> +1 000-000-0000</li>
<li><i class="fa-brands fa-orcid"></i> <a href="https://orcid.org/my-orcid?orcid=0000-0003-4548-1808">0000-0003-4548-1808</a></li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Highly proficient in operating and data collection of a plant breeding program including measurement field phenotypes and integrating genetic data to make selection decision and publish original research</p></li>
<li><p>Highly experienced in processing, analysis, organization, and experimental design of field trials of a novel barley breeding program</p></li>
<li><p>Highly proficient in R statistical software and Excel for data management and analysis. Moderate proficiency in using git version control</p></li>
<li><p>Basic proficiency in Unix shell scripting and command line. Limited experience with Python and Docker.</p></li>
<li><p>Highly proficient in flying unmanned aerial systems for imaging of plant variety trials and breeding populations &amp; Certified FAA UAS Part 107 Remote Pilots License.Â  -present</p></li>
<li><p>Basic proficiency in Agisoft Pro and Open Drone Map software stitching applications.</p></li>
<li><p>Basic proficiency related to chemistry for malting quality analysis</p></li>
<li><p>Demonstrated ability to work in multi-institutional collaborative projects</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>This resume was made with the R package <a href="https://github.com/rstudio/pagedown"><strong>pagedown</strong></a>.</p>
<p>Last updated on 2023-05-05.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Karl Kunze</h2>
<div id="test" class="section level3">
<h3>TEst</h3>
<p>Test</p>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="cornell-university-gradudate-school" class="section level3">
<h3>Cornell University, Gradudate School</h3>
<p>Field of Plant Breeding and Genetics</p>
<p>Ithaca, NY</p>
<p>2023</p>
</div>
<div id="cornell-university-college-of-agriculture-and-life-sciences" class="section level3">
<h3>Cornell University, College of Agriculture and Life Sciences</h3>
<p>B.S. Plant Sciences
Concentration in Plant Breeding and Genetics, minor in Buisness for the Life Sciences</p>
<p>Ithaca, NY</p>
<p>2017</p>
<p>Thesis: A multi-omics study for intra-individual divergence of the distributions between mRNA isoforms in mammals</p>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<div id="graduate-research-assistant" class="section level3">
<h3>Graduate Research Assistant</h3>
<p>Measure components of weed competitive ability in organic naked barley variety trials by using field trait phenotypes and aerial imaging to measure barley vigor and growth over 4 trials and 2 years</p>
<p>N/A</p>
<p>2017 - 2020</p>
<p>Genome wide association studies of organic barley diversity panels across 13 location by year field locations throughout the Northern United Sates</p>
<p>Genetic by environmental analysis of winter naked barley variety trials across 8 environments throughout the Northern United States</p>
<p>Evaluation of dormancy and pre-harvest sprouting across a winter malting barley breeding population consisting of 450 lines over two location and two years. A total of 5 timepoint assays were conducted in 2020 and 8 timepoint assays conducted in 2021.</p>
</div>
</div>
<div id="professional-societies" class="section level2" data-icon="suitcase">
<h2>Professional Societies</h2>
<p>Crop Science Society of America(CSSA) student member -present</p>
<p>National Association of Plant Breeders (NAPB) student member -present</p>
<p>New York State Agriculture Society member -present</p>
<p>Cornell Plant Breeding and Genetics Graduate Student Association, Synapsis -present</p>
<p>Alumni of Alpha Gamma Rho, Professional and Social Agricultural Fraternity, Zeta Chapter</p>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<div id="introduction-to-r-language-for-beginners." class="section level3">
<h3>Introduction to R Language for Beginners.</h3>
<p>Instructor of R and Data Mining Training Courses at SupStat Inc.</p>
<p>Beijing, China</p>
<p>2014</p>
</div>
<div id="computational-biology-and-bioinformatics." class="section level3">
<h3>Computational Biology and Bioinformatics.</h3>
<p>Teaching assistant of GBS CB2-201 courses at UAB</p>
<p>AL, USA</p>
<p>2016 - 2017</p>
</div>
</div>
<div id="selected-publications-and-posters" class="section level2" data-icon="file">
<h2>Selected Publications and Posters</h2>
<div id="genetic-and-epigenetic-signals-are-found-predictive-to-the-distribution-of-intra-individual-divergence-of-alternative-splicing." class="section level3">
<h3>Genetic and epigenetic signals are found predictive to the distribution of intra-individual divergence of alternative splicing.</h3>
<p>Poster for 2013 International Conference of Genomics</p>
<p>/Qingdao, China</p>
<p>2014</p>
<p><strong>Yu L</strong>, Chen B, Zhang Z.</p>
</div>
<div id="escrt-0-complex-modulates-rbf-mutant-cell-survival-by-regulating-rhomboid-endosomal-trafficking-and-egfr-signaling." class="section level3">
<h3>ESCRT-0 complex modulates Rbf mutant cell survival by regulating Rhomboid endosomal trafficking and EGFR signaling.</h3>
<p>J Cell Sci. 2016 May 15;129(10):2075-84.</p>
<p>N/A</p>
<p>2016</p>
<p>Sheng Z, <strong>Yu L</strong>, Zhang T, Pei X, Li X, Zhang Z and Du W.</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
